### DISCUSSION TITLE
StorageModuleElasticsearchProvider watches on wrong files?

### DISCUSSION THREAD

**Discussion by chanjarster** [apache/skywalking#8204] [Q&A] [answered]
I've noticed that `StorageModuleElasticsearchProvider` monitor two files `config.getSecretsManagementFile(), config.getTrustStorePass()` see below:

https://github.com/apache/skywalking/blob/1acba5d2467f6d905a6261e999f85464d5432562/oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/StorageModuleElasticsearchProvider.java#L127-L148

Is the second parameter wrong? `trustStorePass` is not a file path, just a string for password.

**Comment by wu-sheng** [comment, marked as answer]
Yes, this should be removed. The `TrustStorePass` is in the management file.

**Reply by wu-sheng** [reply]
Could you submit a pull request to fix soon?

**Reply by chanjarster** [reply]
No problem, I think should replace `TrustStorePass` with `TrustStorePath`.

**Reply by wu-sheng** [reply]
Yes, it should be.

**Reply by wu-sheng** [reply]
Fixed by https://github.com/apache/skywalking/pull/8205
